pipelines/ocw/pipeline-ocw-www.yml

```diff
@@ -5,23 +5,12 @@ resource_types:
     source:
       repository: mitodl/concourse-rclone-resource
       tag: latest
-  - name: simple-s3
```

Comment:
You still need this resource as it's being used for publishing the compiled site and assets to s3

Reply:
The resource is longer being used for anything, it's been completely replaced by the aws cli docker image task